apps/platform-app/src/app/[locale]/(profile)/_models/rejection-reason.model.ts

```diff
@@ -0,0 +1,11 @@
+import type { RejectionReason } from "@prisma/client"
+
+interface RejectionReasonModel {
+    rejectionReason: string
+}
+
+export function createRejectionReasonModel(data: RejectionReason): RejectionReasonModel {
```

Comment:
**suggestion**: Add a return type to `createRejectionReasonModel` for better type safety.